dashboard/test/factories/factories.rb

```diff
@@ -1691,7 +1691,7 @@
     issuer {"issuer"}
     client_id {"client_id"}
     platform_name {"platform_name"}
-    auth_redirect_url {"auth_redirect_url"}
+    auth_redirect_url {"http://test.org/auth"}
```

Comment:
Was this change necessary or just stylistic update?

Reply:
Rails (or some custom logic) is prefixing the `auth_redirect_url` value unless there is a scheme in the string. 

Example with old string
```
# in factory.rb
auth_redirect_url {"auth_redirect_url"}

# log value of @response.redirect_url in unit test. See how auth_redirect_url string is getting prefixed with the local CDO domain.
https://test-studio.code.orgauth_redirect_url?client_id=client_id ...
```

Example with new string
```
# in factory.rb
auth_redirect_url {"http://test.org/auth"}

# log value of @response.redirect_url in unit test.
http://test.org/auth?client_id=client_id ...
```

This doesn't break any of the tests as they are right now, as I'm only testing the query string params of the redirect url.